tensorboard/plugins/mesh_visualizer/summary_test.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for tensorboard.plugins.mesh_visualizer.summary."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import json
+import tensorflow as tf
+
+from tensorboard.compat.proto import summary_pb2
+from tensorboard.plugins.mesh_visualizer import summary
+from tensorboard.plugins.mesh_visualizer import metadata
+from tensorboard.plugins.mesh_visualizer import plugin_data_pb2
+from tensorboard.plugins.mesh_visualizer import test_utils
+from tensorboard.util import test_util
+
+
+@test_util.run_v1_only('Uses contrib') 
```

Comment:
Does this actually use contrib?  I don't see a call to tf.contrib but I could be missing it.

Ditto for the other tests annotated with `run_v1_only('Uses contrib')`

Reply:
well, maybe the comment is not representative, but I use tf.compat.v1 and I marked all the tests to make sure no tf2 tests are there

Reply:
`contrib` here means something specific, i.e., `tf.contrib.*`. What we are saying here is, the reason for running for v1 should say something more specific like 'uses tf.Session`.